test/integration_test/basic_test.go

```diff
@@ -16,9 +18,58 @@ import (
 )
 
 func TestStorageClusterBasic(t *testing.T) {
+	testutil.TestSpecPath = ""
```

Comment:
if this is test specific, is it better to put in the test func? or in the common test file

Reply:
this is in testutil, is that what you mean by common test file?

Reply:
I mean should we put it in `testInstallMetricsCollector` instead of `TestStorageClusterBasic` as it's only used in that subtest only for now.

Reply:
I intended to put here for all functional tests, unit test has a different path as default and without this it would not work for any functional test.

Reply:
@pure-jliao actually i will rename our current folder specs to testspec so that we keep the same name with unit test and dont have to change the value in code.